### DISCUSSION TITLE
May I ask for help to unbrick the Cam?

### DISCUSSION THREAD

**Discussion by graounest** [roleoroleo/yi-hack-Allwinner-v2#134] [Q&A]
Hi roleoroleo

Many thanks for your great work.

My both cams BFUS 9.0* (y21ga) suddendly bricked yesterday morning.

I tried many things , but I guess I now have to try the unbrick method https://github.com/roleoroleo/yi-hack-Allwinner-v2/wiki/Unbrick-the-cam

Purchasing a ttl/usb right now...

Just wondering a thing about step 4 I don't get : 

4.Open putty with logging enabled.

--> I don't get how to connect to the cam via Putty? Which host? Am I missing something here?

And also is there a way to get a mtdblock4.bin clean file?
I have no factory backup avalaible I dumped before installing the hack (y21ga_0.1.5).
I used this procedure https://github.com/roleoroleo/yi-hack-Allwinner-v2/wiki/Dump-your-backup-firmware-(SD-card)
to get a backup but _after_ the cam was bricked.
Not sure if it is any good though?

Many thanks for your help

**Comment by roleoroleo** [comment]
Before to follow the unbrick procedure, try to unhack the cam
If it doesn't work I will help you.

**Comment by graounest** [comment]
Many thanks for your quick reply,

I indeed also tried the unhack procedure before opening a thread here. Unfortunately it doesn't work either

https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/5403579/unhack_y21ga.tar.gz

I have two cam, bricked at the same time (I suspect a wifi/network malfunction being the origin of the cam brick??)

The cam turns solid orange and nothing more  happens (if I try to start without SD card, with the unhack file, or the v2 0.1.6 for instance - I was on the v 0.1.5 previously when the brick happened - , or if I push the reset button).

**Comment by roleoroleo** [comment]
Here it is:
[mtd4.bin.gz](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/6323908/mtd4.bin.gz)

**Comment by graounest** [comment]
Many Thanks,

I grabbed the file in order to prepare the SD Card , still waiting for the ttl/usb connector.
Just to let me know : as soon as I connect to the cam pins, how can I connect to the cam via Putty? 
Do I have to enter the IP address as a host in putty? I don't get the step 4 "open putty with logging enabled".

I'll try as soon as I get my order. Thanks again roleoroelo

**Reply by roleoroleo** [reply]
When you connect your usb/ttl adapter to the PC, a new serial port interface will appear.
Connect Putty through serial, not through IP.

**Reply by graounest** [reply]
👍 

thanks I'll try, receiving my adapter tomorow hopefully

**Comment by graounest** [comment]
ok, having trouble to connect physically to the cam.

Tried to solder the ttl/usb wires as neatly as possible, but no luck

nvm

**Reply by sphi-nx** [reply]
Hi @graounest did you get this fixed?

I accidentally bricked my IFUS (firmware 9.0.36*) camera. Also, can you provide a link to the ttl/usb cable you used?

Thanks

**Comment by graounest** [comment]
Hi @sphi-nx unfortunately I couldn't manage to get a decent solder on the cam.
I used this ttl https://www.amazon.fr/Yizhet-CP2102-Adaptateur-Convertisseur-pi%C3%A8ces%EF%BC%89/dp/B07TFSZ3ZP/ref=gp_aw_ybh_a_45?_encoding=UTF8&psc=1&refRID=TYNHVP280HYWRWPZK32V

Hope it'll help
